public/components/avatar/avatar.css

```diff
@@ -1,9 +1,59 @@
 @import url(../../main.css);
 
 
-.avatar-section
+.avatar-row
 {
-    padding:5rem 1rem 0;
-    height: 600px;
```

Comment:
Everything looks good to me!! Just one suggestion try to use rem instead of px wherever possible.

Reply:
Yes Anjali Noted. BTW this code with red highlight is not the part of the code, I was using fixed values for the initial prototype. Thanks for Approving.